plugins/modules/vmware_cluster_vcls.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2015, Joseph Callen <jcallen () csc.com>
+# Copyright: (c) 2018, Ansible Project
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+---
+module: vmware_cluster_vcls
+short_description: Override the default vCLS (vSphere Cluster Services) VM disk placement for this cluster.
+description:
+    - Override the default vCLS VM disk placement for this cluster.
+    - Some datastores cannot be selected for vCLS 'Allowed' as they are blocked by solutions as SRM or vSAN maintenance mode where vCLS cannot be configured.
+    - All values and VMware object names are case sensitive.
+author:
+- Joseph Callen (@jcpowermac)
+- Nina Loser (@Nina2244)
+requirements:
+    - Tested on ESXi 7.0
+    - PyVmomi installed.
+options:
+    cluster_name:
+      description:
+      - The name of the cluster to be managed.
+      type: str
+      required: true
+    datacenter:
+      description:
+      - The name of the datacenter.
+      type: str
+      required: true
+      aliases: [ datacenter_name ]
+    allowed_datastores:
+      description:
+        - List of the allowed Datastores.
+        - If there is one more in the current List it will be removed.
+      type: str
```

Comment:
```suggestion
      type: list
      elements: str
```

@Nina2244 In the [CI log](https://cbb3b70ef849cddaa362-d37c8f08686c8ea7c0dbc76419ce2854.ssl.cf1.rackcdn.com/1355/df7f7ea661dbc9dd78f332a178c3ba145252cad7/check/ansible-test-sanity-docker-devel/cb489cf/job-output.txt) there's something about:

```
2022-06-21 09:35:58.938268 | controller | ERROR: Found 2 validate-modules issue(s) which need to be resolved:
2022-06-21 09:35:58.938334 | controller | ERROR: plugins/modules/vmware_cluster_vcls.py:0:0: doc-elements-mismatch: Argument 'allowed_datastores' in argument_spec specifies elements as str,but elements is not documented
2022-06-21 09:35:58.938360 | controller | ERROR: plugins/modules/vmware_cluster_vcls.py:0:0: doc-type-does-not-match-spec: Argument 'allowed_datastores' in argument_spec defines type as 'list' but documentation defines type as 'str'
```

I think defining the type as `list` and adding `elements: str` might fix this.